Automation/create_vnet_script.ps1

```diff
@@ -0,0 +1,26 @@
+# to run this, navigate to the repo and run ./Automation/create_vnet_script.ps1 
+# with the 1st 7 parameters inline -location westus2
+
+param(
+        [string] [Parameter(Mandatory=$true)] $tenantId,
+        [string] [Parameter(Mandatory=$true)] $applicationId,
+        [string] [Parameter(Mandatory=$true)] $secret,
+        [string] [Parameter(Mandatory=$true)] $subscriptionId,
+        [string] [Parameter(Mandatory=$true)] $resourceGroupName,
+        [string] [Parameter(Mandatory=$true)] $location,
+        [string] [Parameter(Mandatory=$true)] $vNetName,
+        [string] [Parameter(Mandatory=$false)] $vNetAddressPrefix
+      )
+
+$pathToVNetTemplate = "./Automation/vnet_template.json"    
+# $pathToVNetTemplate = "D:\VSCode\ad440-winter2021-thursday-repo\Automation\vnet_template.json"  
```

Comment:
I believe this path name has an extra 'Automation'.

Reply:
You're right! Good catch - I need to fix this

Reply:
The script looks great but I believe some comments for the params would help. Like what tenantId and secrete are about.

Reply:
Fixed - I added comments for the applicationId and the secret, but I'm not sure what to say about the tenantId - it's just the tenantId associated with the account, I think.